client/src/app/queries/migration-waves.ts

```diff
@@ -35,7 +36,8 @@ export const useFetchMigrationWaves = () => {
     queryFn: getMigrationWaves,
     refetchInterval: 5000,
     onError: (error) => console.log("error, ", error),
-    onSuccess: () => queryClient.invalidateQueries([TicketsQueryKey]),
+    onSuccess: () =>
+      queryClient.invalidateQueries([TicketsQueryKey, TrackersQueryKey]),
```

Comment:
@ibolton336 does this behave as expected? I think the invalidateQueries array param is intended to be components that have to all be part of a queryKey to invalidate that query, so this should probably be two separate `invalidateQueries` calls for tickets and trackers. Confusing API!

Reply:
Same kind of discussion as https://github.com/konveyor/tackle2-ui/pull/1098/files/83a6edbd8d8db959d1aecd53e1901d213c4cb13d#r1259920085.

"Quick" fix is to have all target queries have a single shared element in the key array and then partial matching do its thing.

FWIW, debugging in to react-query is pretty easy.